flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogTest.java

```diff
@@ -78,4 +99,23 @@ public void testCreateHiveTable() {
                 prop.keySet().stream()
                         .noneMatch(k -> k.startsWith(CatalogPropertiesUtil.FLINK_PROPERTY_PREFIX)));
     }
+
+    @Test
+    public void testRetrieveFlinkProperties() throws TableAlreadyExistException, DatabaseNotExistException, TableNotExistException {
+        ObjectPath hiveObjectPath = new ObjectPath(HiveCatalog.DEFAULT_DB, "testRetrieveProperties");
+
+        Map<String, String> properties = new HashMap<>(new FileSystem().path("/test_path").toProperties());
+
+        properties.put(CatalogPropertiesUtil.IS_GENERIC, String.valueOf(true));
+        properties.put("url", "jdbc:clickhouse://host:port/default");
```

Comment:
set this property to a different value and also verify you can get this property correctly?